tests/python/test_print.py

```diff
@@ -28,3 +25,14 @@ def func(x: ti.i32, y: ti.f32):
         print(x, 1234.5, y)
 
     func(666, 233.3)
+
+
+@ti.archs_excluding(ti.metal, ti.opengl)
+def test_print_string():
+    @ti.kernel
+    def func(x: ti.i32, y: ti.f32):
+        # make sure `%` doesn't break vprintf:
+        print('hello, world! %s %d %f', 233, y)
```

Comment:
What does `%s` here mean?

Reply:
Just to make sure that % doesn't get evaluated in print.
`print('%d', 233)`
will obtain:
`%d 233`

Reply:
Ah I see. Sorry, I misunderstood.